en/middleware/micrortps.md

```diff
@@ -0,0 +1,344 @@
+      {
+          if(m_info.sampleKind == ALIVE)
+          {
+              cout << "\n\n\n\n\n\n\n\n\n\n";
+              cout << "Received sensor_combined data" << endl;
+              cout << "=============================" << endl;
+              cout << "timestamp: " << msg.timestamp() << endl;
+              cout << "gyro_rad: " << msg.gyro_rad().at(0);
+              cout << ", " << msg.gyro_rad().at(1);
+              cout << ", " << msg.gyro_rad().at(2) << endl;
+              cout << "gyro_integral_dt: " << msg.gyro_integral_dt() << endl;
+              cout << "accelerometer_timestamp_relative: " << msg.accelerometer_timestamp_relative() << endl;
+              cout << "accelerometer_m_s2: " << msg.accelerometer_m_s2().at(0);
+              cout << ", " << msg.accelerometer_m_s2().at(1);
+              cout << ", " << msg.accelerometer_m_s2().at(2) << endl;
+              cout << "accelerometer_integral_dt: " << msg.accelerometer_integral_dt() << endl;
+              cout << "magnetometer_timestamp_relative: " << msg.magnetometer_timestamp_relative() << endl;
+              cout << "magnetometer_ga: " << msg.magnetometer_ga().at(0);
+              cout << ", " << msg.magnetometer_ga().at(1);
+              cout << ", " << msg.magnetometer_ga().at(2) << endl;
+              cout << "baro_timestamp_relative: " << msg.baro_timestamp_relative() << endl;
+              cout << "baro_alt_meter: " << msg.baro_alt_meter() << endl;
+              cout << "baro_temp_celcius: " << msg.baro_temp_celcius() << endl;
+
+              // Print your structure data here.
+              ++n_msg;
+              //std::cout << "Sample received, count=" << n_msg << std::endl;
+              has_msg = true;
+
+          }
+      }
+  }
+  ```
+
+> **Note** Normally, for UART transport it's necessary set up the UART port in the Raspberry Pi. To enable the serial port available on Raspberry Pi connector:
+
+1. Make sure the `userid` (default is pi) is a member of the `dialout` group:
+
+  ```sh
+  $ groups pi
+  $ sudo usermod -a -G dialout pi
+  ```
+
+2. You need to stop the already running on the GPIO serial console:
+
+  ```sh
+  $ sudo raspi-config
+  ```
+
+  Go to **Interfacing options > Serial**. Select **NO for *Would you like a login shell to be accessible over serial?*. Valid and reboot.
+
+3. Check UART in kernel:
+
+  ```sh
+  $ sudo vi /boot/config.txt
+  ```
+
+And enable UART setting `enable_uart=1`.
+
+## Hello world and Throughput test
+
+The [Hello world](../middleware/micrortps_hello_world.md) and [Throughput test](../middleware/micrortps_throughput_test.md) show some real-world examples of how to use the features described in this topic. 
+
+## Graphical example of usage
+
+This flow chart shows graphically how the bridge works. It demonstrates a bridge that sends the topic `sensor_combined` from a Pixracer to a Raspberry Pi through UART.
+
+![basic example flow](../../assets/middleware/micrortps/basic_example_flow.png)
```

Comment:
"Almost" best diagram so far. Is the arrow in the middle UART connection to RaPI? If so, great (the image on the right is supposed to be a RaPi. 
Can this have the Agent and Client overlayed?